### ISSUE TITLE
Failed testing Result for matchboxnet and vad_marblenet

### ISSUE THREAD

**Issue by Ryan-ZL-Lin** [dusty-nv/jetson-voice#8]
Hi Dusty
Thanks for the great repo and tutorial of introducing the ASR and NLP on Jetson device. I followed the instructions and got 2 error for 2 model s out of 10., here is my testing result.

----------------------------------------------------
 TEST SUMMARY
----------------------------------------------------
test_asr.py (quartznet)                  PASSED
test_asr.py (quartznet_greedy)           PASSED
test_asr.py (matchboxnet)                FAILED
test_asr.py (vad_marblenet)              FAILED
test_nlp.py (distilbert_qa_128)          PASSED
test_nlp.py (distilbert_qa_384)          PASSED
test_nlp.py (distilbert_intent)          PASSED
test_nlp.py (distilbert_sentiment)       PASSED
test_nlp.py (distilbert_ner)             PASSED
test_tts.py (fastpitch_hifigan)          PASSED

Matchboxnet testing log
----------------------------------------------------
 RUNNING TEST (ASR)
----------------------------------------------------
   model:  matchboxnet
   config: data/tests/asr_keyword.json

binding 0 - 'audio_signal'
   input:    True
   shape:    (1, 64, -1)
   dtype:    DataType.FLOAT
   size:     -256
   dynamic:  True
   profiles: [{'min': (1, 64, 10), 'opt': (1, 64, 150), 'max': (1, 64, 300)}]

binding 1 - 'logits'
   input:    False
   shape:    (1, 12)
   dtype:    DataType.FLOAT
   size:     48
   dynamic:  False
   profiles: []

Vad_marblenet testing log
----------------------------------------------------
 RUNNING TEST (ASR)
----------------------------------------------------
   model:  vad_marblenet
   config: data/tests/asr_vad.json

binding 0 - 'audio_signal'
   input:    True
   shape:    (1, 64, -1)
   dtype:    DataType.FLOAT
   size:     -256
   dynamic:  True
   profiles: [{'min': (1, 64, 10), 'opt': (1, 64, 150), 'max': (1, 64, 300)}]

binding 1 - 'logits'
   input:    False
   shape:    (1, 2)
   dtype:    DataType.FLOAT
   size:     8
   dynamic:  False
   profiles: []

When running command "examples/asr.py --model matchboxnet --wav data/audio/commands.wav", I got an error as follows: 
**_RuntimeError: shape '[1, 154, 2]' is invalid for input of size 79156_**

When running command "examples/asr.py --model vad_marblenet --wav data/audio/commands.wav", I got a similar error like this: 
**_RuntimeError: shape '[1, 34, 2]' is invalid for input of size 17476_**

Have you ever encountered this issue before?
